app/models/match.rb

```diff
@@ -0,0 +1,21 @@
+class Match < ApplicationRecord
+
+  STATUS = ['prepare', 'in game', 'played']
+
+  belongs_to :home_team, class_name: 'Team'
+  belongs_to :invited_team, class_name: 'Team'
+
+  validates :game_name, presence: true, uniqueness: true
+  validates :home_team_score, presence: true, numericality: true
+  validates :invited_team_score, presence: true, numericality: true
+  validates :status, inclusion: { in: STATUS }, allow_nil: true
+  validate  :players_in_team
+
+  def players_in_team
```

Comment:
maybe a more meaningfull name that says what the method validates ? like `user_belongs_to_only_one_team_on_the_match` maybe